src/routes/categories/containers/__tests__/LanguageFailure.spec.js

```diff
@@ -4,10 +4,8 @@ import { Provider } from 'react-redux'
 
 import createHistory from 'modules/app/createHistory'
 import createReduxStore from 'modules/app/createReduxStore'
-import EndpointBuilder from 'modules/endpoint/EndpointBuilder'
-import EndpointProvider from 'modules/endpoint/EndpointProvider'
 
-import LocationModel from 'modules/endpoint/models/LocationModel'
+import LocationModel from 'modules/endpoint/models/CityModel'
```

Comment:
`CityModel` :)

Reply:
Still relevant

Reply:
@maxammann Yep, changed in WEBAPP-234 :D